src/core/link.h

```diff
@@ -11,10 +11,14 @@ struct link {
   struct {
     struct counter *txbytes, *rxbytes, *txpackets, *rxpackets, *txdrop;
   } stats;
+  char pad[24];
   // Two cursors:
   //   read:  the next element to be read
   //   write: the next element to be written
-  int read, write;
+  int read;
+  char read_pad[60];
+  int write;
+  char write_pad[60];
```

Comment:
I'm thinking that 56 bytes of padding should be enough here when `int` is 8 bytes. However we should probably be more explicit and declare `read` and `write` as either `int64_t` or `int32_t`. (I have a vague recollection that Mike Pall discourages the use of `uint64_t` with FFI for some reason -- must understand that properly one day.)

Generally LuaJIT does understand declarations like `__aligned__((64))` but I think C only supports that on structs/arrays.